### PR TITLE
Fix create mutations

### PR DESCRIPTION
**Pull request by wtrocki**
Rebased version from @ardatan

### CODE REVIEW

**Comment by wtrocki** [comment]
This works end to end with graphql cli giving us full typesafety

File: packages/graphback-codegen-resolvers/src/formatters/ApolloTsResolverFormatter.ts [hunk 1]
```diff
@@ -46,13 +46,13 @@ const generateTypeResolvers = (context: TargetResolverContext, name: string, opt
   let resolverType = '';
   let typedImports = ''
   if (options.types) {
```
Comment (wtrocki) [inline packages/graphback-codegen-resolvers/src/formatters/ApolloTsResolverFormatter.ts]: We do not have this done for our own CLI so this code can be tested with graphql cli. 
We can add that later.

**Comment by wtrocki** [comment]
Index is still not valid

**Comment by wtrocki** [comment]
Fixed index issue. Custom resolvers are still generated the old way.

File: packages/graphback-codegen-client/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@graphback/codegen-client",
   "version": "0.10.0-rc2",
-  "description": "GraphQL-Code-Generator compatible client side queries generator",
+  "description": "GraphQL-Code-Generator compatibile client side queries generator",
```
Comment (craicoverflow) [inline packages/graphback-codegen-client/package.json]:
```suggestion
  "description": "GraphQL-Code-Generator compatible client side queries generator",
```

**Pull request merged**